### PR TITLE
Fixes #10074 - OSTree CLI changes

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: lib/hammer_cli_katello/repository.rb [hunk 1]
```diff
@@ -70,6 +70,10 @@ class InfoCommand < HammerCLIKatello::InfoCommand
           field :docker_manifest_total, _("Docker Manifests"), Fields::Field, :hide_blank => true
           field :docker_tag_total, _("Docker Tags"), Fields::Field, :hide_blank => true
         end
+        collection :ostree_branches, _("RPM OS Tree Branches"),
```
Comment (parthaa) [inline lib/hammer_cli_katello/repository.rb]: Can ya change this to OSTree Branches to be consistent with UI

Comment (parthaa) [inline lib/hammer_cli_katello/repository.rb]: Also
```
$ hammer repository info --id=25
ID:                   25
Name:                 marquez
Label:                marquez
Organization:         Default Organization
Red Hat Repository:   no
Content Type:         ostree
URL:                  http://sparta.usersys.redhat.com:2025
Publish Via HTTP:     no
Published At:         https://galaxy.example.com/pulp/ostree/web/Default_Organization-ostree-marquez
Relative Path:        Default_Organization/Library/custom/ostree/marquez
Product:              
    ID:   1
    Name: ostree
Sync:                 
    Status: Error
Created:              2016/03/04 14:22:43
Updated:              2016/03/04 14:22:47
Content Counts:       

RPM OS Tree Branches: 
    fedora-atomic/f21/x86_64/updates-testing/docker-host
    fedora-atomic/f21/x86_64/updates/docker-host
```
I see 2 branches there but no value for content unit counts. Might need to fix that to accept ostree_branch

Comment (parthaa) [inline lib/hammer_cli_katello/repository.rb]: May be for each branch do
```
Branch: ..
Version: ..
Commit: ..
Date: ..
``
like what we show here -> https://partha.fedorapeople.org/ostree-prs/PR-5858/08-Manage-Branches-Repo.png  (actually that UI screen shot got updated.. Now also shows us commit and date)

File: lib/hammer_cli_katello/ostree_branch.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        o.expand.including(:products, :organizations, :content_views)
+      end
+    end
+
+    class InfoCommand < HammerCLIKatello::InfoCommand
+      output do
+        field :id, _("ID")
+        field :name, _("Name")
+        field :version, _("Version")
+        field :commit, _("Commit")
```
Comment (parthaa) [inline lib/hammer_cli_katello/ostree_branch.rb]: add ```field :version_date, _("Date")``` to be consistent with what will be shown in the UI

**Comment by parthaa** [comment]
The outputs look great. ACK pending my suggestions .

One other thing, we  need to improve the output for content view version. I have filed a redmine issue for that http://projects.theforeman.org/issues/14167 .  I feel we should be showing the ostree branches that belong to a content view version to match  https://partha.fedorapeople.org/ostree-prs/PR-5874/02-content-view-version-branches.png.  But  we'd need => https://github.com/Katello/katello/pull/5874/files for that. Conclusion being I 'll update issue 14167 and we'll track that separately.

File: lib/hammer_cli_katello/repository.rb (hunk 1 above)
Comment (daviddavis) [inline lib/hammer_cli_katello/repository.rb]: I think that would be too long? If people want to get that info, they should use the ostree-branch command.

Comment (daviddavis) [inline lib/hammer_cli_katello/repository.rb]: Wondering if I should just hide this field and show content counts like the other content types. Thoughts?

File: lib/hammer_cli_katello/ostree_branch.rb (hunk 2 above)
Comment (daviddavis) [inline lib/hammer_cli_katello/ostree_branch.rb]: Done.

**Comment by daviddavis** [comment]
@parthaa I probably wouldn't list the branches for a content view version but other than that, that seems fine.

**Comment by parthaa** [comment]
ack

**Pull request merged**